custom_components/opnsense/device_tracker.py

```
@@ -35,257 +35,196 @@
 
 
-class OPNsenseScannerEntity(OPNsenseEntity, ScannerEntity):
+class OPNsenseScannerEntity(OPNsenseEntity, ScannerEntity, RestoreEntity):
     """Represent a scanned device."""
 
     def __init__(
         self,
-        hass: HomeAssistant,
         config_entry: ConfigEntry,
         coordinator: OPNsenseDataUpdateCoordinator,
         enabled_default: bool,
```

Ruff

Boolean-typed positional argument in function definition

(FBT001)